src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -690,21 +690,24 @@ configure(struct sched_cmd *cmd)
 	}
 	write_unlock(&zdev->attr_rwlock);
 
-	/* Enable interrupt from host to PS when new commands are ready */
-	if (exec->cq_interrupt) {
-		/* Stop CQ check thread */
-		if (zdev->exec->cq_thread)
-			kthread_stop(zdev->exec->cq_thread);
-
-		/* At this point we are good. No one is polling CQ */
-		cq_irq = zdev->ert->irq[ERT_CQ_IRQ];
-		ret = request_irq(cq_irq, sched_cq_isr, 0, "zocl_cq", zdev);
-		if (ret) {
-			DRM_WARN("Failed to initial CQ interrupt. "
-				 "Fall back to polling\n");
-			exec->cq_interrupt = 0;
-			exec->cq_thread = kthread_run(cq_check, zdev, name);
-		}
+	if(zdev->ert)
```

Comment:
Thanks for fixing this, could we do it a little bit neat with other coding style, like
if (zdev->ert) {
    ...
}

Reply:
Thanks for pointing this out. Fixed it.